app/tests/version/test_read_test_harness_backend_version.py

```diff
@@ -20,7 +20,8 @@
 import pytest
 
 from app import utils
-from app.core.config import settings
+# TODO: MATTER_SETTINGS
+from test_collections.matter.config import matter_settings
```

Comment:
We can load the module dynamically and only test the SDK version if the matter module exists.
what do you think?